README.md

```diff
@@ -59,14 +63,12 @@ class Restaurant < ActiveRecord::Base
     has_many :dishes, through: "Restaurant__c"
     belongs_to :chef, through: %w(Chef__c Cuisine__c)
 
+    converts_with StyleAdapter
```

Comment:
Should `converts_with` perhaps take a list of adapters (one or more)? This would let each adapter be fairly specific (as our current adapters are) and be helpful where we currently do many conversions.

It would _also_ allow us to always insert our default adapter into the chain (or let others opt-in to using it, if we're not keen on mandating it). I imagine that even if we're using an adapter for something specific, we still may want the date/time formatting of that default adapter.